packages/dds/tree/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -0,0 +1,181 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { FieldKey, EmptyKey } from "../../tree";
+import { IEditableForest, TreeNavigationResult, ITreeSubscriptionCursor } from "../../forest";
+import { TreeSchemaIdentifier } from "../../schema-stored";
+
+export const proxySymbol = Symbol.for("editable-tree");
+
+/**
+ * A dummy placeholder for a Proxy target.
+ */
+export interface IEditableTree {}
```

Comment:
This should provide a more specific type. Something more like:

```suggestion
export interface IEditableTree {
    [type]: TreeSchema;
    [proxySymbol]: ObjectEditableTree; // Implementation looks like it claims to have this
    [Symbol.iterator]: Iterator<IEditableTree>; // Not sure this is the right type
    [key: string]: undefined | IEditableTree;
}
```

Reply:
Still not sure on how the structure, tbd